From 72963d5d984e8874f55638941e92b2b34939a8bb Mon Sep 17 00:00:00 2001
From: Henry Chen <henry.chen@oss.cipunited.com>
Date: Wed, 9 Oct 2024 15:26:25 +0800
Subject: [PATCH 38/39] util/grub-install: mips64el-efi: rename removable
 executable

as loongson3 machines with 64-bit EFI only accept BOOTMIPS.EFI
---
 util/grub-install.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util/grub-install.c b/util/grub-install.c
index fabba39fc..3fc2a6007 100644
--- a/util/grub-install.c
+++ b/util/grub-install.c
@@ -1293,7 +1293,7 @@ main (int argc, char *argv[])
 	      efi_file = "BOOTRISCV64.EFI";
 	      break;
 	    case GRUB_INSTALL_PLATFORM_MIPS64EL_EFI:
-	      efi_file = "BOOTMIPS64EL.EFI";
+	      efi_file = "BOOTMIPS.EFI";
 	      break;
 	    default:
 	      grub_util_error ("%s", _("You've found a bug"));
-- 
2.47.0

